app/components/Feedback/index.tsx

```diff
@@ -39,48 +40,69 @@ const Feedback = ({
   upHint,
   downHint,
   options,
+  formClassName,
   onSubmit,
 }: FeedbackProps) => {
-  const [showFeedback, setShowFeedback] = useState(false)
+  const [showThanks, setShowThanks] = useState(false)
   const [showFeedbackForm, setShowFeedbackForm] = useState(false)
+  const [vote, setVote]: any = useState(undefined)
+
+  const thanksRef = useRef<HTMLDivElement | null>(null)
 
   useEffect(() => {
-    const timeout = setInterval(() => setShowFeedback(false), 6000)
-    return () => clearInterval(timeout)
-  }, [showFeedback])
+    if (showThanks && thanksRef.current) {
+      thanksRef.current.style.opacity = '1'
```

Comment:
use classes for css. Or contitional rendering if you want to use js. Don't mix the two - there lies madness

Reply:
why this way, rather than the previous approach?

Reply:
You right on both accounts. I've updated the code